pkg/utils/emails/templates/verify_email.html

```diff
@@ -0,0 +1,14 @@
+<p>Hello {{ .FullName }},</p>
+
+<p>Thank you for signing up for Ensign! Please verify your email address by clicking <a href="{{ .VerifyURL }}">this
+        link</a>.</p>
+
+<p>If you cannot click the link, please copy and paste the following URL into your browser:</p>
+
+<p>{{ .VerifyURL }}</p>
+
+If you are having trouble verifying your email address, please contact us at <a
+    href="mailto:support@rotational.io">support@rotational.io</a>.
+
+<p>Thank you,<br />
+    The Ensign Team</p>
```

Comment:
Did @eschmier review the copy for the welcome email? E.g. is this going to be our standard sign-off for emails?

Reply:
I've tagged you and @eschmier for review on the welcome email PR. I put this in as a starting point but I'm assuming it will change.

Reply:
Ok - I think it's a little easier to review the copy and style for all the emails at once; so let's create a story for @eschmier when we're done with the initial batch of emails.